### PR TITLE
Ledger analytics and JSONRenderer Enhancement

### PR DESCRIPTION
**Pull request by blueorbitz**
- Introduce Ledger analytic as part of XRP hackathon. Details refer [here](
https://devpost.com/software/insight-block-for-xrp)
- Addressing Issue #3

### CODE REVIEW

**Comment by WietseWind** [comment]
Thank you for building and contributing @blueorbitz!

I just approved deploying this to test build URLs at Netlify. End result, for example:

https://deploy-preview-6--hooks-v2-explorer.netlify.app/tx/FB645709F9205CBDEF933E16B0461B99680FE33F9501BD8146914880190397A3

I went through the code & deployment. Before merging, I have a couple of thoughts/requests/suggestions:

1. The Memo decoding: how can I get that to work? I tried in the preview, but fields matching your definition (contains `memo`, HEX value) don't seem to be clickable/hoverable/...
2. The ledger analytics (cool!) allows for range loading. I think that's going to result in unwanted effects. I noticed you show a confirmation dialog if users want to load a lot of ledgers, but people are going to, and they are going to click Yes anyway. This puts a lot of load on public nodes & it will make the site and their browser unusable. I would for example ask them to give one ledger and allow to +5 and -5 from that (small range) to prevent a bad user & network load experience.
3. If you build the project (`npm run build`) there are quite some code standard warnings. I'd like you to address those 😇 
4. When you select (pin) multiple single ledgers this error pops up every ledger when re-calculating the stats:
image

So, TL;DR: thank you! very cool :) I hope you can find the time to incorporate the above.

**Comment by blueorbitz** [comment]
@WietseWind thanks for the quick review and suggestion. Let me work on those over the weekend then.
Might expect some delay due to my time commitment.

**Comment by blueorbitz** [comment]
> The Memo decoding: how can I get that to work? I tried in the preview, but fields matching your definition (contains memo, HEX value) don't seem to be clickable/hoverable/...

It is a clickable event, due to the limited functionality of the native library.  To use it you need to enable the JsonRenderer with ` :useToastAction="true" `.
```
 <JsonRenderer v-if="selectedTxs.length" :data="selectedTxs[txViewIndex]" :useToastAction="true" />
```

Alternatively, I can fix with by making the useToastAction as the default action.

**Comment by smman89** [comment]
@WietseWind, Hi! This project is alive?

**Comment by WietseWind** [comment]
> @WietseWind, Hi! This project is alive?

Definitely, live over here: https://explorer.xrplf.org/
